pkg/store/bucket_test.go

```diff
@@ -3342,3 +3341,80 @@ func TestBucketIndexReader_decodeCachedPostingsErrors(t *testing.T) {
 		testutil.NotOk(t, err)
 	})
 }
+
+func TestBucketStore_LabelAPIs(t *testing.T) {
+	t.Cleanup(func() { custom.TolerantVerifyLeak(t) })
+
+	testLabelAPIs(t, func(tt *testing.T, extLset labels.Labels, appendFn func(app storage.Appender)) storepb.StoreServer {
```

Comment:
tt is the testing.T from the particular subtest run; we need it because we dont want to skip all tests and also because we want to close resources properly